### ISSUE TITLE
Blocked from sending BTC to yourself

### ISSUE THREAD

**Issue by brandonmarshall-tm** [blockstack/blockstack-app#5349]
A user was trying to send BTC to themselves to consolidate UTXOs, but Leather blocked them from doing so.

It might better to give a warning that you are sending to yourself, and still allow you to proceed, versus blocking the user from doing it entirely. There are real use cases where one may want to send to themselves.

![Screenshot 2024-05-07 at 9 20 07 AM](https://github.com/leather-wallet/extension/assets/122313287/7499a591-c872-421d-80bd-5db7d27f6926)

**Comment by brandonmarshall-tm** [comment]
Upping this to a P2 as there are no clear alternatives other than using a different wallet or paying more.

As BTC fees climb, users will find opportunities to consolidate UTXOs during times of low fees. We should not block users from sending to themselves.

**Comment by pete-watters** [comment]
> Upping this to a P2 as there are no clear alternatives other than using a different wallet or paying more.
> 
> As BTC fees climb, users will find opportunities to consolidate UTXOs during times of low fees. We should not block users from sending to themselves.

Can users just create a new account on the wallet and send funds there? That would consolidate their UTXOs albeit at a different address. They can send it to themselves, just not to the same address

**Comment by brandonmarshall-tm** [comment]
Yes technically correct, just double the fees required if they want to keep the same wallet. It just feels like an arbitrary rule we have in place when a warning should suffice

**Comment by pete-watters** [comment]
> Yes technically correct, just double the fees required if they want to keep the same wallet. It just feels like an arbitrary rule we have in place when a warning should suffice

Agreed and it could even be a nice feature to offer in its own right - UTXO consolidation

**Comment by pete-watters** [comment]
I've removed validation that was blocking users from sending BTC to their own address in [this PR](https://github.com/leather-io/extension/pull/6085). I've kept the validation for STX as that network does not support it.

This means users will now be able to send BTC to themselves and this consolidate their UTXOs
